src/EditorFeatures/Core/CodeActions/CodeActionEditHandlerService.cs

```diff
@@ -342,11 +341,9 @@ internal class CodeActionEditHandlerService : ForegroundThreadAffinitizedObject,
                         {
                             var editorWorkspace = workspace;
                             var navigationService = editorWorkspace.Services.GetRequiredService<IDocumentNavigationService>();
-                            var location = await navigationService.GetLocationForSpanAsync(
-                                editorWorkspace, documentId, resolvedRenameToken.Span, cancellationToken).ConfigureAwait(false);
 
-                            if (location != null &&
-                                await location.NavigateToAsync(NavigationOptions.Default, cancellationToken).ConfigureAwait(false))
+                            if (await navigationService.TryNavigateToSpanAsync(
+                                    this.ThreadingContext, editorWorkspace, documentId, resolvedRenameToken.Span, cancellationToken).ConfigureAwait(false))
```

Comment:
Indenting intentional this way?

Reply:
yup.  i dont' like my arguments aligning with the parent, they are better indented to visually tell tehy're children, not siblings :)